HDSAviz/basic_plot.py

```diff
@@ -0,0 +1,122 @@
+"""This is a basic script for plotting an analysis file."""
```

Comment:
I will reformat my code so you can replace this section with an import statement